src/Jackalope/Transport/Logging/DebugStack.php

```diff
@@ -28,6 +28,13 @@ class DebugStack implements LoggerInterface
     public $enabled = true;
 
     /**
+     * Show the debug backtrace for each call
+     *
+     * @var boolean
+     */
+    public $backtrace = false;
```

Comment:
It could be. I was folllowing the example of `enabled` above, which I guess is `public` for convenience...